### ISSUE TITLE
[TS] selectAction title should be passed to UI automation for accessibility

### ISSUE THREAD

**Issue by andrewleader** [Microsoft/AdaptiveCards#2429]
**Automatically generated task. Any updates to the title and body will be overwritten. Do NOT modify the labels.**

TS implementation of #1298

**Comment by andrewleader** [comment]
Closing, confirmed TS already does this (sets `aria-label` and tested in Narrator and it reads as "{title} button".
